docs/examples/index.js

```diff
@@ -543,6 +580,8 @@ myChart.setOption(themeManager.getChartOptions());
 
   // Display the chart using the configuration items and data just specified.
   myChart.setOption(themeManager.getChartOptions());
+
+  window.setTimeout(myChart.resize);
```

Comment:
This change should be removed after merge of https://github.com/Orange-OpenSource/ods-charts/pull/250